lib/__tests__/standalone-cache.test.js

```diff
@@ -0,0 +1,146 @@
+"use strict"
+
+const path = require("path")
+const standalone = require("../standalone")
+const hash = require("../utils/hash")
+const fixturesPath = path.join(__dirname, "fixtures")
+const fsExtra = require("fs-extra")
```

Comment:
I'd rather we avoid this new dependency and just use regular `fs`. For copying you could use https://github.com/sindresorhus/cpy. Additionally, we should be able to avoid sync calls for all of this because Jest has good async/Promise support.